billboard.py

```diff
@@ -179,7 +185,10 @@ def fetchEntries(self):
         dateElement = soup.select_one('button.chart-detail-header__date-selector-button')
         if dateElement:
             dateText = dateElement.text.strip()
-            self.date = datetime.datetime.strptime(dateText, '%B %d, %Y').strftime('%Y-%m-%d')
+            curDate = datetime.datetime.strptime(dateText, '%B %d, %Y')
+            if self.date and curDate < datetime.datetime.strptime(str(self.date), '%Y-%m-%d'):
+                raise ValueError('Date argument is after the date of the latest issue')
```

Comment:
Can you add a comment here (in the code) explaining the reasoning behind this? Something like, "For future dates, Billboard.com returns a valid webpage containing no data but displaying the date of the latest issue."